src/main/java/androidx/core/view/View.kt

```diff
@@ -286,3 +288,26 @@ inline fun <reified T : ViewGroup.LayoutParams> View.updateLayoutParams(block: T
     block(params)
     layoutParams = params
 }
+
+/**
+ * Sets the Outline to the view defined by the corner radius.
+ *
+ * @see Outline.setRoundRect
+ */
+@RequiresApi(21)
+inline  fun View.round(@Px radius: Float) {
```

Comment:
This seems a little too specific. The name of the method isn't great either.

Reply:
Ok I'll close it